### ISSUE TITLE
Category URL Rewrite page not working

### ISSUE THREAD

**Issue by dexterityzx** [magento/magento2#4483]
## Steps to reproduce
1. Create a category with single quotes in its name, for example, Mother's day, Valentine's day.
2. Go to Marketing => URL rewrites
3. Select "For category"
## Expected result
1. System displays a category tree and we can select and edit the url rewrites.
## Actual result
1. The category tree is empty and the navigation tab in the left is not working either.

Suspicious reason : The parsing of the json object - categoryTree does not escape the single quote and make the json object close without }.

![capture](https://cloud.githubusercontent.com/assets/13892193/15225319/f349d858-184b-11e6-9e83-d1e908d7725b.PNG)

**Comment by maksek** [comment]
@dexteritycc, seems like issue with JS files on you env, maybe related to static content. Have you tried other pages, let's say with grids (product, cms)?

**Comment by quienti** [comment]
https://github.com/magento/magento2/issues/3677

I'm actually with Magento 2.0.5 and It's work fine

**Comment by dexterityzx** [comment]
@maksek ,the other pages works fine.
@quienti, the newly installed one is ok, but the one I upgraded from older version is not working.

**Comment by dexterityzx** [comment]
@maksek 
I found that the js fails when the category's name contains single quote. Here is how the json("categoryTree") ends without the closing }

`{"id":59,"parent_id":36,"children_count":0,"is_active":true,"name":"Valentine`

it should be Valentine's Day in the "name." 

The category tree is displayed after I deleted all the single quote in the category name. Is there any way to escape the single quote?
